ndlarflow/h5_to_root_ndlarflow.py

```diff
@@ -36,6 +36,10 @@ def main(argv=None):
     if (len(sys.argv)>2):
         if (int(sys.argv[2])==1):
             useData=True
+    output="/exp/dune/data/users/rdiurba/flowToROOT"
```

Comment:
Recommend something like the following to get rid of your username here:

```
username=os.getenv("USER")
output="/exp/dune/data/users/"+username+"/flowToROOT"
```

Reply:
It would also be good to add a comment near the top of the main() function to say what run arguments are accepted:

`python h5_to_root_ndlarflow,py inputFile isData (0 or 1) outputDir`